src/hosting/normalizedHostingConfigs.ts

```diff
@@ -56,6 +56,41 @@ function filterOnly(configs: HostingConfig[], onlyString: string): HostingConfig
   return filteredConfigs;
 }
 
+function filterExcept(configs: HostingConfig[], exceptString: string): HostingConfig[] {
+  if (!exceptString) {
+    return configs;
+  }
+
+  const exceptTargets = exceptString.split(",");
+  if (exceptTargets.includes("hosting")) {
+    return [];
+  }
+
+  const exceptSet = new Set(
```

Comment:
Nit: Don't love this name either - its redundant with the type, and it doesn't convey that this is only the hosting excepts. Consider something like `hostingExcepts` instead?

Reply:
Done.